src/app/static/src/app/localStorageManager.ts

```diff
@@ -14,7 +14,8 @@ const stripDataFromProject = (project: Project): Project => {
     return {
         ...project,
         currentRegion: stripDataFromRegion(project.currentRegion),
-        regions: project.regions.map(stripDataFromRegion)
+        regions: project.regions.map(stripDataFromRegion),
+        strategies: []
```

Comment:
Should strategies go into `serialiseState` as well to persist them?

Reply:
Stripped these deliberately as they're not user input - I can't think of a good reason to serialise them. They use up localStorage, don't take long to regenerate and if anything I rather encourage re-generation in case we identify any bugs in the optimiser.